.mailmap

```diff
@@ -0,0 +1,107 @@
+Chris Allan <callan@glencoesoftware.com> <callan@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Carlos Neves <carlos@glencoesoftware.com>
+Colin Blackburn <c.blackburn@dundee.ac.uk> <C.Blackburn@dundee.ac.uk>
+Colin Blackburn <c.blackburn@dundee.ac.uk> <colin@ximenes.org.uk>
+Colin Blackburn <c.blackburn@dundee.ac.uk> <cblackburn@05709c45-44f0-0310-885b-81a1db45b4a6>
+Curtis Rueden <ctrueden@wisc.edu> <ctrueden@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+David Whitehurst <david@glencoesoftware.com> <david@05709c45-44f0-0310-885b-81a1db45b4a6>
+Dominik Lindner <d.lindner@dundee.ac.uk>
+Dominik Lindner <d.lindner@dundee.ac.uk> <dominikl@users.noreply.github.com>
+Donald MacDonald <dzmacdonald@lifesci.dundee.ac.uk>
+Donald MacDonald <dzmacdonald@lifesci.dundee.ac.uk> <dzmacdonald@05709c45-44f0-0310-885b-81a1db45b4a6>
+Donald MacDonald <dzmacdonald@lifesci.dundee.ac.uk> <donald@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Donald MacDonald <dzmacdonald@lifesci.dundee.ac.uk> <donald@05709c45-44f0-0310-885b-81a1db45b4a6>
+Donald MacDonald <dzmacdonald@lifesci.dundee.ac.uk> <donald@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Donald MacDonald <dzmacdonald@lifesci.dundee.ac.uk> <donald@Grumpy.local>
+Donald MacDonald <dzmacdonald@lifesci.dundee.ac.uk> <donald@Grumpy.dyn.lifesci.dundee.ac.uk>
+Douglas Creager <douglas.creager@comlab.ox.ac.uk> <dcreager@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Douglas Russell <douglas.russell@bioch.ox.ac.uk>
+Douglas Russell <douglas.russell@bioch.ox.ac.uk> <douglas_russell@hms.harvard.edu>
+Emil Rozbicki <emil@glencoesoftware.com> <emilroz@gmail.com>
+Harry Hochheiser <harryh@pitt.edu> <hsh@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Helen Flynn <h.flynn@dundee.ac.uk> <omehelen@gmail.com>
+Jean-Marie Burel <j.burel@dundee.ac.uk>
+Jean-Marie Burel <j.burel@dundee.ac.uk> <jburel@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Jean-Marie Burel <j.burel@dundee.ac.uk> <jburel@05709c45-44f0-0310-885b-81a1db45b4a6>
+Jean-Marie Burel <j.burel@dundee.ac.uk> <jburel@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Jean-Marie Burel <j.burel@dundee.ac.uk> <jburel@users.noreply.github.com>
+Jeff Mellen <jeffm@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Jesse Corrington <jesse@glencoesoftware.com> <jesse.corrington@gmail.com>
+Josh Moore <josh@openmicroscopy.org> <j.a.moore@dundee.ac.uk>
+Josh Moore <josh@openmicroscopy.org> <josh@glencoesoftware.com>
+Josh Moore <josh@openmicroscopy.org> <josh@example.com>
+Josh Moore <josh@openmicroscopy.org> <josh.moore@gmx.de>
+Josh Moore <josh@openmicroscopy.org> <josh@05709c45-44f0-0310-885b-81a1db45b4a6>
+Josh Moore <josh@openmicroscopy.org> <jmoore@05709c45-44f0-0310-885b-81a1db45b4a6>
+Josh Moore <josh@openmicroscopy.org> <jmoore@f08c0cc7-402c-0410-8265-edbaffadc9ce>
+Josh Moore <josh@openmicroscopy.org> <jmoore@6f2cb1de-eb0d-0410-b157-e593188b5901>
+Joshua Ballanco <jballanc@glencoesoftware.com>
+Joshua Ballanco <jballanc@glencoesoftware.com> <jballanc@gmail.com>
+Kenneth Gillen <k.h.gillen@dundee.ac.uk>
```

Comment:
```
  7  Kenneth Gillen <k.h.gillen@dundee.ac.uk>
3  Kenny Gillen <k.h.gillen@dundee.ac.uk>
```
Should this not be combined using 
```Proper Name <proper@email.xx> Commit Name <commit@email.xx>```

Reply:
Since the same email addresses is used, I think

```
Proper Name <commit@email.xx>
```

is sufficient to map to the canonical real name - https://git-scm.com/docs/gitmailmap